Plugins/Power Manager/Misc Scripts/PowerShell/Refresh_Inventory.ps1

```diff
@@ -0,0 +1,175 @@
+  
```

Comment:
This doesn't seem to be PMP specific - probably a better candidate for Core functionality